templates/vue/src/components/lightbox/H5PIframe.vue

```diff
@@ -245,23 +234,11 @@ export default {
                 h5pIframeComponent.handlePause(h5pIframeComponent.node)
                 break
               }
-
-              case h5pObj.Video.BUFFERING: {
-                const { id, mediaType } = h5pIframeComponent.node
-                thisTapestryTool.updateMediaIcon(id, mediaType, "loading")
-                break
-              }
             }
           })
           if (h5pIframeComponent.autoplay) {
             setTimeout(() => {
               h5pVideo.play()
-              thisTapestryTool.recordAnalyticsEvent(
-                "app",
-                "auto-play",
-                "h5p-video",
-                h5pIframeComponent.node.id
-              )
```

Comment:
Why has this been removed?

Reply:
I removed all references to `thisTapestryTool` to avoid any errors, but I think I have yet to add this functionality back in. Thanks for pointing it out!